src/sentry/static/sentry/app/views/health/list/healthTableHead.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import {
+  StyledLayout,
```

Comment:
Do we need all of these Styled prefixes for these components? We generally just use them if we are styling another component, otherwise, it makes it a bit harder to read.